src/plugins/irc/irc-ctcp.c

```diff
@@ -1050,11 +1215,12 @@ irc_ctcp_recv_dcc (struct t_irc_server *server, const char *nick,
             {
                 weechat_infolist_new_var_string (item, "plugin_name", weechat_plugin->name);
                 weechat_infolist_new_var_string (item, "plugin_id", server->name);
-                weechat_infolist_new_var_string (item, "type_string", "file_recv");
+                weechat_infolist_new_var_string (item, "type_string", "file_recv_active");
                 weechat_infolist_new_var_string (item, "filename",
                                                  (filename) ? filename : pos_file);
                 weechat_infolist_new_var_integer (item, "port", atoi (pos_port));
                 weechat_infolist_new_var_string (item, "start_resume", pos_start_resume);
+                weechat_infolist_new_var_string (item, "token", pos_token);
```

Comment:
`token` field should be an integer here:

```C
                weechat_infolist_new_var_integer (item, "token", atoi (pos_token));
```

Reply:
Done.